unity/Holo/Assets/Scripts/model_with_plate/ModelWithPlate.cs

```diff
@@ -21,8 +22,8 @@ public class ModelWithPlate : MonoBehaviour, IClickHandler
     public CompoundButton ButtonTranslate;
     public CompoundButton ButtonRotate;
     public CompoundButton ButtonScale;
-    public Color ButtonActiveColor = new Color(0f, 0.90f, 0.88f);
-    public Color ButtonInactiveColor = new Color(1f, 1f, 1f);
+    static public Color ButtonActiveColor = new Color(0f, 0.90f, 0.88f);
+    static public Color ButtonInactiveColor = new Color(1f, 1f, 1f);

```

Comment:
Setting these fields as `static` makes them not configurable in Unity Editor.

Instead I would leave them as non-static, and leave `SetButtonState` method as non-static, and inside `ModelClippingPlaneControl` I would call it like `GetComponent<ModelWithPlate>().SetButtonState(...)`.

Reply:
done